.github/workflows/github-actions.yml

```diff
@@ -116,82 +92,64 @@ jobs:
+        # Temporary settings to use same username and password as travis ci
         run: |
           mysql -e 'CREATE USER 'travis'@'localhost';' -uroot -proot
           mysql -e 'GRANT ALL PRIVILEGES ON * . * TO 'travis'@'localhost';' -uroot -proot
           mysql -e 'FLUSH PRIVILEGES;' -uroot -proot
 
-      - name: Cleanup Gradle Cache
-          # Remove some files from the Gradle cache which might have cached by earlier builds.
-        run: |
-          rm -f ~/.gradle/caches/modules-2/modules-2.lock
-          rm -f ~/.gradle/caches/modules-2/gc.properties
-
-      - name: Cache Gradle packages
-        uses: actions/cache@v2
+      - uses: burrunan/gradle-cache-action@v1
+        name: Run integration tests
         with:
-          path: |
-            ~/.gradle/caches
-            ~/.gradle/wrapper
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
-          restore-keys: |
-            ${{ runner.os }}-gradle-
-
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
+          job-id: jdk11
+          arguments: --scan intTest codeCoverageReport
+          gradle-version: wrapper
 
-      - name: Integration Tests
-        run: ./travis-int-test.sh
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
 
   publish:
 
     runs-on: ubuntu-latest
     needs: [ unit-test, store-test, int-test ]
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
     env:
       ARTIFACTORY_USER: ${{ secrets.ARTIFACTORY_USER }}
       ARTIFACTORY_API_KEY: ${{ secrets.ARTIFACTORY_API_KEY }}
       GIT_SECRET: ${{ secrets.GIT_SECRET }}
-      PUBLISH: ${{ secrets.PUBLISH }}
     steps:
       - name: Checkout Ambry
         uses: actions/checkout@v2
         # Full fetch depth is used to fetch all existing tags in the repo to assign a version for this build
         with:
           fetch-depth: 0
-        # environment variables are only supported in step conditionals
-        # TODO once the PUBLISH variable is no longer needed move the if statement up to the job level
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' && env.PUBLISH == 'true'}}
 
       - name: Set up JDK 11
         uses: actions/setup-java@v2
         with:
           java-version: '11'
           distribution: 'adopt'
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' && env.PUBLISH == 'true'}}
 
-      - name: Cleanup Gradle Cache
-          # Remove some files from the Gradle cache which might have cached by earlier builds.
-        run: |
-          rm -f ~/.gradle/caches/modules-2/modules-2.lock
-          rm -f ~/.gradle/caches/modules-2/gc.properties
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' && env.PUBLISH == 'true'}}
+      - name: Publish To JFrog artifactory
+        run: ./travis-publish.sh
```

Comment:
Typo for removing publishing command.

Reply:
Thanks for catching this! I will remove it.